Makefile

```diff
@@ -135,7 +135,7 @@ run: manifests generate fmt vet binary ## Run a controller from your host.
 # (i.e. docker build --platform linux/arm64). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
-docker-build: ## Build docker image with the manager.
+docker-build: manifests generate fmt vet ## Build docker image with the manager.
```

Comment:
Should this also include `build`?

Reply:
No, `build` is for building outside a container, so isn't needed for the `docker-build`

Reply:
Oh right, sorry ... forgot that's handled in the Dockerfile.